README.md

```diff
@@ -1,31 +1,15 @@
 # Gazebo ROS Demos
 
-* Author: Dave Coleman <davetcoleman@gmail.com>
+* Author: Alejandro Hernández
```

Comment:
I'd be in favor of simply removing this line... we're both authors, but so are many others who have contributed to this repo.